xlsx_reader.py

```diff
@@ -1,191 +0,0 @@
-        magazines_list.append(magazine_properties)
-    return magazines_list
-
-
-# writing authors, books and copies data in database
-def get_books(file_location):
-    books_properties = get_book_data(file_location)
-    asset_codes = []
-
-    for book in books_properties:
-        title = book['title']
-        asset = book['asset']
-        asset_codes.append(asset)
-        authors = book['authors']
-        list_of_authors = []
-
-        if isinstance(authors, tuple):
-            authors_id = []
-            first_name = str(authors[0])
-            last_name = str(authors[1])
-            author = create_library_item(db.session, Author,
-                                         last_name=last_name,
-                                         first_name=first_name)
-            id_of_auth = author.id
-            authors_id.append(id_of_auth)
-            list_of_authors.append(author)
-            book = create_library_item(db.session, Book, title=title)
-            book.authors.append(author)
-
-        elif isinstance(authors, list):
-            authors_id = []
-            for auth_name in authors:
-                f_name = str(auth_name[0])
-                l_name = str(auth_name[1])
-                author = create_library_item(db.session, Author,
-                                             last_name=l_name,
-                                             first_name=f_name)
-                id_of_auth = author.id
-                authors_id.append(id_of_auth)
-                list_of_authors.append(author)
-                book = create_library_item(db.session, Book,
-                                           title=title)
-                book.authors.append(author)
-                # checking if asset value is an empty string
-                # which violates uniqueness of the asset code
-                if not asset:
-                    create_library_item(db.session, Copy,
-                                        library_item_id=book.id,
-                                        library_item=book)
-                else:
-                    create_library_item(db.session, Copy,
-                                        library_item_id=book.id,
-                                        library_item=book,
-                                        asset_code=asset)
-
-
-# writing magazine's data in database
-def get_magazines(file_location):
-    magazines_properties = get_magazine_data(file_location)
-    for i in magazines_properties:
-        title = i['title']
-        issue = str(i['issue'])
-        year = str(i['year'])
-        if not year:
-            print('Magazine ', title, issue, ' has no year information.')
-        else:
-            year = datetime.strptime(str(int(i['year'])), '%Y')
-            create_library_item(db.session, Magazine,
-                                title=title,
-                                year=year,
-                                issue=issue)
```

Comment:
omg I just had a stroke because of these changes, but discovered you reverted this in your next commit 🥇